extensions/control-plane/data-plane-transfer/data-plane-transfer-client/src/main/java/org/eclipse/dataspaceconnector/transfer/dataplane/DataPlaneTransferClientExtension.java

```diff
@@ -66,16 +68,20 @@ public String name() {
 
     @Override
     public void initialize(ServiceExtensionContext context) {
+        var monitor = context.getMonitor();
+
         DataPlaneTransferClient client;
         if (dataPlaneManager != null) {
             // Data plane manager is embedded in the current runtime
             client = new EmbeddedDataPlaneTransferClient(dataPlaneManager);
+            monitor.info(() -> format("Using %s.", EmbeddedDataPlaneTransferClient.class.getSimpleName()));
```

Comment:
Please remove this message, it does not provide useful operation information

Reply:
It should be useful to detect unintensional setup in the first place [as DataPlaneInstanceClientExtension  do](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/93f53eb917fd5ffc0670a909bb05dd92fd542c42/extensions/data-plane-selector/data-plane-selector-client/src/main/java/org/eclipse/dataspaceconnector/dataplane/selector/DataPlaneInstanceClientExtension.java#L58-L67).

Reply:
updated log level.

Reply:
Can you either remove this or change the message emitted to something more useful? If a message is emitted, it should be a debug message with something like "Using embedded Data Plane", not the classname. Thx.

Reply:
I updated the message.